content/join/policy.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Policy
+parent: Joining the infrastructure
+nav_order: 10
+---
+
+# Policy
+
+In order to ensure proper testbed operation and maintenance we require that all sites (external and internal) abide by the policies outlined below.
+
+1. Provide one or more dedicated machines on-site to act as the SCIONLab nodes. The machine should not be used for any other tasks beyond its role as an SCIONLab node.
+2. Provide root (or sudo) access to the SCIONLab management institution (currently ETH Zurich).
+3. Allow the SCIONLab management institution to install, remove and update SCIONLab related software as needed for the operation and maintenance of the network.
+4. Work with the SCIONLab management institution to solve any firewall and tunneling issues that may arise during the installation, configuration and operation of the SCIONLab node(s).
+5. Provide the name, email and telephone information of an easily and quickly reachable site-specific operator who will be the main contact whenever issues arise with the local SCIONLab node(s). A backup operator, while not required, is highly encouraged. Operator email addresses must belong to the institution. All local operators will be required to join and regularly monitor an SCIONLab community mailing list and are required to respond to requests from the testbed management institution in a reasonable time, but no more than 3 working days.
+6. Ensure that the machine has limited logins beyond the SCIONLab account. A local operator account is highly encouraged, but general user accounts should not be present on the SCIONLab node. In addition, only SCIONLab-related software should be allowed to run on the machine.
```

Comment:
Is this actually true? Don't we want to allow e.g. active researchers that do stuff with SCIONLab to have access to the machines? (Maybe not, because researchers mess things up :D but if not, then who exactly is the target group for joining the infra?)

Reply:
This is very true. If they want to play with the machine, they should be using user AS. I do not trust any of this dude to do anything inside this machine and not breaking stuff

Reply:
Okay, but:
* we say "[blablabla] you are committed to do research with SCION, and using user ASes is not enough for your plans" in instructions.md
* we say "Connecting to the testbed for the sake of being connected is a violation of the current policies" in policy.md

So that sounds to me like we want people to join the infra when they want to do fancy research with it.

And then we say "limited logins beyond the SCIONLab account" here, so that they cannot do fancy research with it :D

So, it seems to me that these three things together strongly limit who is supposed to be joining the infra and why. The only use case that I can come up with while satisfying all three is just something like "we really really want a local attachment point". Is this actually the only use case? If no, please tell me about another one :D And if yes, should we just say that explicitly at the beginning of `instructions.md`?

Reply:
Okay, let me give you a specific example. We have a SCIONLab node somewhere in Austria (I believe TU Vienna), where some "Charlie dude" logs into the machine and changes configuration of the SCION services to something super dumb, making the services fail hard during the start-up. As this is not transparent for the rest of the network, I do not want him to be allowed to misconfigure stuff. By this I mean the configuration of the infrastructure should be stable and deterministic and someone ssh-ing and modifying random files contradicts both these points.

Reply:
Let me give you a second example. We used to have a node in Israel which was not used by anyone for an unknown amount of time. Once this node has died (i.e. went no-contact without anybody informing us) it took these guys more than 3 months to reply to Juan's email. I need a mechanism to enforce a strong reaction in these cases as I do not want SCIONLab to become a graveyard of machines which used to work once and nothing more.